bats/fb-content-katello.bats

```diff
@@ -172,3 +196,8 @@ EOF
   skip # because of above
   puppet agent --test && grep -q Lorem /tmp/dummy
 }
+
+#@test "Delete an Organization" {
```

Comment:
Kinda - I'm not sure the deletion is actually working properly despite "passing" this test :/ 

@jlsherrill @parthaa any idea if organization delete actually works today?